spec/imsc-hrm.html

```diff
@@ -205,10 +202,6 @@
         <li>at least 1 content producing implementation and at least 1 validating implementation; or</li>
         <li>at least 2 independent validating implementations.</li>
       </ul>
-      <p>The Working Group has identified the following <dfn data-cite="w3c-process/#at-risk">at
-      risk</dfn> features:</p>
-
-      <div class="issue" data-number="63"></div>
```

Comment:
Is it worth replacing this with an explicit statement that there are no at-risk features? It's a bit strange to have a lone "At risk features may be removed..." sentence without explaining that there aren't any.

Reply:
I looked at IMSC 1.2 CR, which did not have at-risk features, and could not find any mention of them, so I think we should just remove the sentence "At risk features may be removed...".

Reply:
I believe this is up to editors on at-risk text, whether both to remove following line for possible removal of at-risk feature, and to have an explicit statement with no at-risk...

(bikeshed has `At Risk` keyword, and all of at-risk related text will appear only when `At Risk` keyword is included in its source.)